api_backend.yaml

```diff
@@ -0,0 +1,14 @@
+swagger: "2.0"
```

Comment:
we should not have the specs in this library, let's see how we can avoid them

Reply:
fixed in c5dd8fc